.github/workflows/docs.yml

```diff
@@ -28,13 +28,18 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: '3.10'
-          cache: 'pip'
 
-      - name: Upgrade pip
-        run: python -m pip install --upgrade pip
+      - name: Install the latest version of uv
+        uses: astral-sh/setup-uv@v4
+        with:
+          version: "latest"
+          enable-cache: true
+          cache-dependency-glob: "requirements**.txt"
```

Comment:
Ok, good for me, dependencies are (still) not a lot, so caching not that relevant

Reply:
Ok, I still would like to have a requirements.in / uv.lock file to be used for both caching and releases:

https://docs.astral.sh/uv/guides/projects/#uvlock

> uv.lock is a cross-platform lockfile that contains exact information about your project's dependencies.  Unlike the pyproject.toml which is used to specify the broad requirements of your project, the lockfile contains the exact resolved versions that are installed in the project environment. This file should be checked into version control, allowing for consistent and reproducible installations across machines.

And removing this defaults to uv.lock, instead of the toml:

https://github.com/astral-sh/setup-uv/tree/v4/?tab=readme-ov-file#cache-dependency-glob

I have just fixed pushing the lock (uv lock)